servicemesh-release-notes/topics/rn-fixed-issues.adoc

```diff
@@ -8,10 +8,4 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-BZ# https://bugzilla.redhat.com/show_bug.cgi?id=000001[000001]
-
-There was a werewolf in the last release.  You would have only encountered this werewolf if it was a full moon and you were outside.  The werewolf has been shot with a silver bullet and no longer exists.
-
-JIRA# https://issues.jboss.org/browse/MAISTRA-5[000005]
-
-There was an unrecognized vampire in the last release.  As a result, the vampire bit a lot of people and sired multiple vampires.  But after the liberal application of sunlight, garlic, and holy water, all vampires have been removed from the current release and you no longer need to wear turtlenecks when going outside.
+https://github.com/Maistra/openshift-ansible/pull/19/[MASITRA-21] -  The default in the installer of 512Mi was too low for tracing.  Increased default Elasticsearch memory from 512 Mb to 1Gb.
```

Comment:
@vikram-redhat, are the GitHub references in this topic and the next topic ok? 

512 MB and 1 GB